CMakeLists.txt

```diff
@@ -119,6 +119,7 @@ endif()
 include(CheckCXXCompilerFlag)
 
 set(COMMON_CXX_FLAGS
+  -std=c++11
```

Comment:
You don't need to do that, it's already done l. 38 by ` set(CMAKE_CXX_STANDARD 11)`